src-js/app.js

```diff
@@ -1,11 +1,15 @@
 import * as Cookie from 'js-cookie';
 import * as search from './instantsearch.js';
 import * as lang from './lang.js'
-import OptimizelyClient from './optimizelyClient.js';
+import OptimizelyClient from './optimizely.js';
+import AnalyticsClient from "./analytics.js";
+
 import * as highlightJSBadge from 'highlightjs-badge';
 
 window.Cookie = Cookie;
 search.init();
 lang.init();
 
-window.OptimizelyClient = new OptimizelyClient();
\ No newline at end of file
+window.OptimizelyClient = new OptimizelyClient();
+window.AnalyticsClient = AnalyticsClient;
```

Comment:
if we want to keep this pseudo-singleton pattern, perhaps we should add a comment here explaining why we're not creating a new instance?

Reply:
Yeah because it only has static methods, `AnalyticsClient` should not be instantiated. The only reason it is here is so webpack catches it and includes it in the `app.build.js`. I'll add a comment but in the near future (when we move everything to webpack) we should make it work so webpack automatically catches everything that is in `src-js/`